docs/Migrating.md

```diff
@@ -13,10 +19,13 @@
      in the `Brain` GameObjects
   *  Agents have a `Brain` field in the Inspector, you need to drag the
   appropriate Brain asset in it.
```

Comment:
Brain asset -> `Brain` Scriptable Objects

Since you referred to them as Scriptable Objects before.